user_docs/en/userGuide.md

```diff
@@ -42,15 +42,15 @@ Major highlights include:
 
 #### Minimum System Requirements {#MinimumSystemRequirements}
 
-* Operating Systems: all 32-bit and 64-bit editions of Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2012 R2.
+* Operating Systems: all 32-bit and 64-bit editions of Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2016.
   * both AMD64 and ARM64 variants of Windows are supported, including Copilot+ PCs.
-  * Note that 32-bit operating systems are no longer under active support.
-  * Note that Windows 8.1 and Windows Server versions older than 2022 are no longer under active support.
+  * Note that 32-bit operating systems are no longer under active support (Windows 10 is the last 32-bit operating system).
+  * Note that Windows 10 (all versions) and Windows Server versions older than 2025 are no longer under active support.
```

Comment:
Are you sure that server 2022 is no longer actively supported? I think this is a bit premature.

Reply:
>Are you sure that server 2022 is no longer actively supported? I think this is a bit premature.


>Technically, it is premature to say that Windows Server 2022 is not under active support. But as far as GitHub Actions goes, Server 2025 is the image NV Access uses (Server 2022 is supported until 2031).

When we say "active support" in the user guide we mean support by NV Access, not Microsoft.

Reply:
Also note mainstream support for 2022 ends 2026/10/13. I think it's likely we will continue some level of support for Server 2022 until mainstream support ends. However, since we're only building and testing on Win 11 / Server 2025 now, we should only recommend that.